### ISSUE TITLE
Update MinIO client to 8.4.3

### ISSUE THREAD

**Issue by lazystone** [quarkiverse/quarkus-minio#144]
There is a related PR from Dependabot - #137 

Problem is that new MinIO client requires newer OkHttp dependency.

In gradle.kts this can be worked around as

```kotlin
    implementation(enforcedPlatform("com.squareup.okhttp3:okhttp-bom:4.10.0")) {
        because("MinIO requires OkHttp >= 4.8.1")
    }
    implementation(enforcedPlatform("com.squareup.okio:okio-bom:3.2.0")) {
        // https://github.com/square/okhttp/blob/parent-4.10.0/build.gradle#L20
        because("OkHttp")
    }
```

**Comment by jtama** [comment]
This can't unfortunately be done as Quarkus enforce okhttp3, and won't migrate to the new version of okhttp wich brings much more classe.
See https://github.com/quarkusio/quarkus/issues/25876

Keeping the issue opened as migration will be done as soon as possible.

**Comment by jtama** [comment]
this will soon be released for Quarkus 3.x branch, and we need to wait for 2.17 upstream to release for 2.X (https://github.com/quarkusio/quarkus/pull/30480)

**Comment by jtama** [comment]
So I released an updated version of the minio extension, but the dependency os still enforced by the plateform so I have to rollback...

**Comment by jtama** [comment]
Closed by #272
